src/Storefront/Resources/app/storefront/src/plugin/zoom-modal/zoom-modal.plugin.js

```diff
@@ -70,7 +70,9 @@ export default class ZoomModalPlugin extends Plugin {
 
     init() {
         this._triggers = this.el.querySelectorAll(this.options.triggerSelector);
```

Comment:
You can't remove variables. Someone could rely on them

Reply:
Okaaaay. But people should not rely on private marked funkctions and variables 🙊